hubblestack/extmods/modules/win_pulsar_winaudit.py

```diff
@@ -218,11 +218,25 @@ def process(configfile='salt://hubblestack_pulsar/hubblestack_pulsar_win_config.
                             if leftover.startswith(os.sep):
                                 _append = False
         if _append and config_found:
+            r['Hash'] = _get_item_hash(r['Object Name'], config.get('checksum', 'sha256'))
```

Comment:
we will caculate the hash of only those files which are to be monitored as per our pulsar config file